package/batocera/emulators/flycast/flycast.mk

```diff
@@ -35,7 +35,7 @@ endif
 
 # RPI: use the legacy Broadcom GLES libraries
 ifeq ($(BR2_PACKAGE_BATOCERA_TARGET_RPI2)$(BR2_PACKAGE_BATOCERA_TARGET_RPIZERO2),y)
-    FLYCAST_CONF_OPTS += -DUSE_VIDEOCORE
+    FLYCAST_CONF_OPTS += -DUSE_VIDEOCORE=ON
```

Comment:
rpi2 is using mesa3d
https://github.com/batocera-linux/batocera.linux/blob/master/configs/batocera-rpi2_defconfig#L76

Reply:
Okay. I wonder who set it to use videocore. I'll make another build and send it to the tester to see if there's any performance difference.